api/client/swarm/join_token.go

```diff
@@ -12,11 +12,6 @@ import (
 	"golang.org/x/net/context"
 )
 
-const (
-	flagRotate = "rotate"
-	flagQuiet  = "quiet"
-)
```

Comment:
actually, I removed them entirely at first for the same reason, and just put literal strings in `flags.BoolVar(...`. Then I noticed that most other flags were defined as a constant in api/client/swarm/opts.go.

If you prefer, I can change to the literal string